lib/xcode.js

```diff
@@ -185,5 +198,21 @@ const getMaxIOSSDK = _.memoize(
   }
 );
 
+function clearInternalCache () {
+
+  // memoized functions
+  const memoized = [getPath, getVersion, getAutomationTraceTemplatePath,
+                    getMaxIOSSDK];
+
+  memoized.forEach( (f) => {
+    if (f.cache) {
+      f.cache = new _.memoize.Cache();
+    }
+  });
+
```

Comment:
I'm bothered by es7's new import/export syntax. I wanted to define all the exported functions as an array like this:

```
exports = [func0, func1, func2];
export exports;
```
because then I could loop over the exported functions  to clear their cache.

But export syntax doesn't allow this.